packages/nextjs/test/config/webpack/webpackPluginOptions.test.ts

```diff
@@ -0,0 +1,157 @@
+      },
+      silent: false,
+      debug: true,
+      sentryUrl: 'my-url',
+      sourcemaps: {
+        assets: ['my-asset'],
+        ignore: ['my-ignore'],
+      },
+      release: {
+        name: 'my-release',
+        create: false,
+        finalize: false,
+        dist: 'my-dist',
+        vcsRemote: 'my-origin',
+        setCommits: {
+          auto: true,
+        },
+        deploy: {
+          env: 'my-env',
+        },
+      },
+    });
+
+    expect(generatedPluginOptions.authToken).toBe('my-auth-token');
+    expect(generatedPluginOptions.debug).toBe(true);
+    expect(generatedPluginOptions.headers).toStrictEqual({ 'my-test-header': 'test' });
+    expect(generatedPluginOptions.org).toBe('my-org');
+    expect(generatedPluginOptions.project).toBe('my-project');
+    expect(generatedPluginOptions.reactComponentAnnotation?.enabled).toBe(true);
+    expect(generatedPluginOptions.release?.create).toBe(false);
+    expect(generatedPluginOptions.release?.deploy?.env).toBe('my-env');
+    expect(generatedPluginOptions.release?.dist).toBe('my-dist');
+    expect(generatedPluginOptions.release?.finalize).toBe(false);
+    expect(generatedPluginOptions.release?.name).toBe('my-release');
+    expect(generatedPluginOptions.release?.setCommits?.auto).toBe(true);
+    expect(generatedPluginOptions.release?.vcsRemote).toBe('my-origin');
+    expect(generatedPluginOptions.silent).toBe(false);
+    expect(generatedPluginOptions.sourcemaps?.assets).toStrictEqual(['my-asset']);
+    expect(generatedPluginOptions.sourcemaps?.ignore).toStrictEqual(['my-ignore']);
+    expect(generatedPluginOptions.telemetry).toBe(false);
+    expect(generatedPluginOptions.url).toBe('my-url');
+
+    expect(generatedPluginOptions).toMatchObject({
+      authToken: 'my-auth-token',
+      debug: true,
+      headers: {
+        'my-test-header': 'test',
+      },
+      org: 'my-org',
+      project: 'my-project',
+      reactComponentAnnotation: {
+        enabled: true,
+      },
+      release: {
+        create: false,
+        deploy: {
+          env: 'my-env',
+        },
+        dist: 'my-dist',
+        finalize: false,
+        inject: false,
+        name: 'my-release',
+        setCommits: {
+          auto: true,
+        },
+        vcsRemote: 'my-origin',
+      },
+      silent: false,
+      sourcemaps: {
+        assets: ['my-asset'],
+        ignore: ['my-ignore'],
+      },
+      telemetry: false,
+      url: 'my-url',
+    });
+  });
+
+  it('returns the right `assets` and `ignore` values when run on the server', () => {
+    const buildContext = generateBuildContext({ isServer: true });
+    const generatedPluginOptions = getWebpackPluginOptions(buildContext, {});
+    expect(generatedPluginOptions.sourcemaps).toMatchObject({
+      assets: ['/my/project/dir/.next/server/**', '/my/project/dir/.next/serverless/**'],
+      ignore: [],
+    });
+  });
+
+  it('returns the right `assets` and `ignore` values when run on the client', () => {
```

Comment:
just to clarify, do you mean with "when run on the client" that client bundles are generated? Just wanna avoid confusion here 😅

Reply:
Correct, I'll update the wording!